src/Game/CMakeLists.txt

```diff
@@ -9,4 +9,6 @@ target_sources(uking PRIVATE
   gameScene.h
   gameStageInfo.cpp
   gameStageInfo.h
+  debugInput.cpp
```

Comment:
Could you move this to KingSystem (ksys)? Looks like DebugInput is initialised from ksys::InitializeApp, which means it is part of the framework part of BotW (`src/KingSystem`), not the game part (`src/Game`). Not sure where it should go -- maybe just put it in `KingSystem/ksysDebugInput.cpp` (unless you see a better subfolder in KingSystem?)